### PR TITLE
add permissions to comply with 1.8+

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

File: app/controllers/api/v2/compliance/policies_controller.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 module Api::V2
   module Compliance
     class PoliciesController < ::Api::V2::BaseController
+      include Foreman::Controller::SmartProxyAuth
+
+      add_smart_proxy_filters :content
```
Comment (ares) [inline app/controllers/api/v2/compliance/policies_controller.rb]: I don't see this feature defined in smart_proxy_openscap, isn't content feature added by pulp node? Should we  have our own feature perhaps?

Comment (shlomizadok) [inline app/controllers/api/v2/compliance/policies_controller.rb]: content feature isn't defined by pulp. yet you are right - it should be similar to https://github.com/theforeman/foreman_openscap/blob/master/app/controllers/api/v2/compliance/arf_reports_controller.rb#L22
fixing. Thanks :+1:

**Comment by ares** [comment]
LGTM

**Comment by ares** [comment]
Tested, fixes the unauthorized issue when downloading content file, thanks @shlomizadok

**Pull request merged**